### ISSUE TITLE
`treehouses message slack` lacks custom message for slack commands

### ISSUE THREAD

**Issue by JLKwong** [treehouses/cli#2115]
`treehouses message slack` should echo something like:
> Please use `treehouses message slack apitoken` to get started with Slack.

**Comment by dogi** [comment]
@JLKwong yes ... custom message based on status of configuration of apitoken
